winkelmand.php

```diff
@@ -2,23 +2,32 @@
 include __DIR__ . "/components/header.php";
 include __DIR__ . "/helpers/utils.php";
 ?>
+    <h2>Winkelmandje</h2><br>
+<?php
+if(isset($_POST["mislukt"])){
+    print ("<h3 style='color: red'>betaling mislukt probeer het opnieuw</h3>");
```

Comment:
Je kan dit een class geven en dan op die class in de winkelmand.css deze styling zetten